### ISSUE TITLE
[Module 6: Multimedia] Learning outcome on media players

### ISSUE THREAD

**Issue by carlosapaduarte** [w3c/wai-curricula#527]
Should we include a learning outcome about identifying accessibility requirements for media players? Or do we consider this out of scope for content authors?

**Comment by daniel-montalvo** [comment]
Hey @carlosapaduarte 

I would say identify works, as long as we weave it in so that it does not conflict with  #493 

The question would then be who is responsible for applying such requirements. If we agree it is the developer, then we have an issue in the developers part as this was probably not clear enough in the "Rich applications" module, where I think it would fit best.

**Comment by carlosapaduarte** [comment]
I agree with you that it is the developer's responsibility to enforce the requirements. Can't we update the developers module?

**Comment by carlosapaduarte** [comment]
Agreed!

**Comment by daniel-montalvo** [comment]
Hey @carlosapaduarte 

Marking this for future iteration based on
https://www.w3.org/2022/05/17-wai-curricula-minutes.html
